bundle/manifests/webhook-operator.clusterserviceversion.yaml

```diff
@@ -167,12 +179,8 @@ spec:
     url: https://your.domain
   version: 0.0.1
   webhookdefinitions:
-  - admissionReviewVersions:
-    - v1beta1
-    - v1
-    containerPort: 443
-    targetPort: 4343
-    deploymentName: webhook-operator-webhook
+  - admissionReviewVersions: null
```

Comment:
I guess this explains why I'm seeing an error when deploying my operator :-)

Reply:
This [is fixed](https://github.com/operator-framework/operator-sdk/pull/3903) in version 1.0.1 of the operator SDK.

Reply:
This one seems to have missed the v1.0.1 train.

Reply:
https://github.com/operator-framework/operator-sdk/commit/88a6f988c8d85ab686b802ad524dbd823f3227d9

Reply:
My mistake - the commit isn't in v1.0.1 of the SDK.